model-optimizer/extensions/load/tf/loader.py

```diff
@@ -103,10 +104,10 @@ def load(self, graph: Graph):
 
         # try to detect layout from the nodes of the graph. If there are no convolution nodes in N(D)HWC layout then we
         # consider that the graph is in NCHW layout and no layout conversion should be performed
-        if not argv.disable_nhwc_to_nchw and not argv.silent and not graph_or_sub_graph_has_nhwc_ops(graph):
-            log.error('The TensorFlow model does not contain Convolution operations with N(D)HWC layout. Most likely '
-                      'the model should be converted using additional "--disable_nhwc_to_nchw" command line parameter '
-                      'which disables model layout conversion inside the Model Optimizer.', extra={'is_warning': True})
+        if not argv.disable_nhwc_to_nchw and not graph_or_sub_graph_has_nhwc_ops(graph):
+            if not argv.silent:
+                log.debug('disable_nhwc_to_nchw" was automatically enabled.')
+            for_graph_and_each_sub_graph_recursively(graph, update_cmd_params_and_layout)
```

Comment:
I think that this pass should be called if `argv.disable_nhwc_to_nchw` is set.

Reply:
What will be `disable_nhwc_to_nchw` is set and the graph contains Conv layers?
What will be `disable_nhwc_to_nchw` is set and the graph does not contain Conv layers?

Reply:
> I think that this pass should be called if argv.disable_nhwc_to_nchw is set as well shouldn't be?
It wasn't called before, don't think we want to change behavior of legacy option.
Model most likely will not convert if `disable_nhwc_to_nchw` is set and the graph contains Conv layers. If model doesn't have Conv layers and `disable_nhwc_to_nchw` is enabled the behavior will be the same as without option, with only one change, old_api_map will NOT include transposing back to initial layout. So that users that used that option will have same layout as they used to have in old API.